### PR TITLE
improve documentation for get.samples 

### PR DESCRIPTION
**Pull request by dlebauer**
## Description

**very minor fix**

Clarified use of get.samples - either for random samples or specific quantiles

## Motivation and Context

User expressed confusion over use

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/priors/R/priors.R [hunk 1]
```diff
@@ -182,15 +182,28 @@ pr.samp <- function(distn, parama, paramb, n) {
...
 ##'
-##' Like pr.samp, with prior as a single input
+##' Similar to the prior sample function \link{pr.samp}, except 1) it takes the prior as a named dataframe
+##' or list and it can return either a random sample of length n OR a sample from a quantile specified as p
 ##' @title Get Samples
-##' @param prior data.frame with distn, parama, paramb
-##' @param n number of samples to return
-##' @param p probability vector, pre-generated upstream to be used in the quantile function
+##' @param prior data.frame with distn, parama, and optionally paramb.
+##' @param n number of samples to return from a random sample of the rdistn family of functions (e.g. qnorm)
```
Comment (istfer) [inline modules/priors/R/priors.R:189]:
```suggestion
##' @param n number of samples to return from a random sample of the rdistn family of functions (e.g. rnorm)
```

**Review by istfer — APPROVED** [review]
just one small change suggestion

File: modules/priors/R/priors.R [hunk 2]
```diff
@@ -182,15 +182,28 @@ pr.samp <- function(distn, parama, paramb, n) {
...
+##' @param p vector of quantiles from which to sample the distribution; typically pre-generated upstream
+##' in the workflow to be used by the qdist family of functions (e.g. qnorm)
```
Comment (istfer) [inline modules/priors/R/priors.R:191]:
```suggestion
##' in the workflow to be used by the qdistn family of functions (e.g. qnorm)
```

**Comment by robkooper** [comment]
@dlebauer can you comment on the recommendations from istem.

**Comment by dlebauer** [comment]
@robkooper I've accepted @istfer 's suggestions

**Pull request merged**